### ISSUE TITLE
Kane's Serta I Comfort exclusion from all discount codes

### ISSUE THREAD

**Issue by kbower** [kbower/tickettest1#1103]
_Imported from trac ticket 1103.
Created by **kbower**
Opened in trac: Sep 29, 2011, 10:14:33 AM
Last modified in trac: Oct 6, 2011, 4:13:55 PM_
***
Kent or Rob,

Good morning, I would like to add the following skus to standard exclusion list for all existing and new discount codes where ITM.VE_CD = 'SERT' and ITM.FAMILY_CD in ('95050', '99246')?

Thanks,

*Wayne Liburd*
----
Note: this is a simple addition to this rule function
```python
    def universally_excluded():
        if product.obsoleteid == u'S':
            return True
        if product.vendor in (u'TPED',u'STEA',):
            return True
        return False
```

**Comment by kbower** [comment]
_**kbower** trac comment on Sep 29, 2011, 10:25:44 AM:_

This rule should apply to all discounts universally, even those that are not special discounts in anyway and exist in GERS?

**Comment by kbower** [comment]
_**kbower** trac comment on Sep 29, 2011, 10:36:55 AM:_

Kent,

Yes, that is correct.

Thanks,

*Wayne Liburd*

**Comment by kbower** [comment]
_**kbower** trac comment on Sep 30, 2011, 7:57:44 AM:_

(Changeset [[1760]](../commit/af12a4874775f3dced255144e3a40a2acfe0879e))
 * Kane's Serta I Comfort exclusion from all discount codes (closes #1103)

**Comment by kbower** [comment]
_**kbower** trac status change on Sep 30, 2011, 7:57:44 AM:_ **closed**

**Comment by kbower** [comment]
_**kbower** trac comment on Oct 3, 2011, 11:20:19 AM:_

Kent,

Good morning, I have a correction to this exclusion list. It should be where ITM.MNR_CD = '500' and ITM.VE_CD = 'SERT' and   ITM.FAMILY_CD in ('33719', '33721', '33738', '33722', '33723').

Thanks,

*Wayne Liburd
*


**
**

**Comment by kbower** [comment]
_**kbower** trac status change on Oct 3, 2011, 11:21:08 AM:_ **reopened**

**Comment by kbower** [comment]
_**kbower** trac comment on Oct 6, 2011, 4:13:55 PM:_

(Changeset [[1788]](../commit/eabc5279b99d32eb6f0b80504e2ec413c04365ad))
 * fixes #1103 again (correction from Wayne)

**Comment by kbower** [comment]
_**kbower** trac status change on Oct 6, 2011, 4:13:55 PM:_ **closed**
